panda/src/navigation/navMeshNode.h

```diff
@@ -0,0 +1,21 @@
+#ifndef NAVMESHNODE_H
+#define NAVMESHNODE_H
+
+#include <string>
+#include "geom.h"
+#include "pandaFramework.h"
+#include "pandaSystem.h"
+#include "navMesh.h"
+#include <string>
+
+class NavMeshNode: public PandaNode
+{
+private:
+  NavMesh _nav_mesh;
+
+public:
+  NavMeshNode(std::string &name, NavMesh nav_mesh);
```

Comment:
Should be `const std::string &name` since it's not modifying it

NavMesh should be a reference-counted class, so `PT(NavMesh)` (and make NavMesh inherit from ReferenceCount, or better yet, TypedWritableReferenceCount).

Reply:
Done in latest commit.